Xamarin.PropertyEditing.Mac/Controls/Custom/CommandButton.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Windows.Input;
+using AppKit;
+
+namespace Xamarin.PropertyEditing.Mac.Controls.Custom
+{
+	public class CommandButton : NSButton
+	{
+		private ICommand command;
+
+		public ICommand Command
+		{
+			get { return this.command; }
+			set
+			{
+				if (this.command != null)
+					this.command.CanExecuteChanged -= CanExecuteChanged;
+
+				this.command = value;
+				this.command.CanExecuteChanged += CanExecuteChanged;
+			}
+		}
+
+		public CommandButton ()
+		{
+			Activated += (object sender, EventArgs e) => {
+				this.command?.Execute (null);
```

Comment:
Avoid ref cycle caused by the lambda capture.

```suggestion
				var button = (CommandButton)sender;
				button.command?.Execute (null);
```